test/local/senders/sms.js

```diff
@@ -34,16 +34,8 @@ const sendSms = sinon.spy((from, to, message, callback) => {
     ]
   })
 })
-let nexmoBalance = 1
```

Comment:
Even though we don't refer to it, that property is returned from nexmo for `sendSms`. I'm minded to keep it around so that the mock result stays as close as possible to the real thing.